target.go

```diff
@@ -0,0 +1,82 @@
+package main
+
+import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"strings"
+
+	"github.com/tsuru/tsuru/cmd"
+)
+
+type targetSet struct{}
+
+func (targetSet) Info() *cmd.Info {
+	return &cmd.Info{
+		Name:    "target-set",
+		Usage:   "target-set <target>",
+		Desc:    "sets the remote tranor server",
+		MinArgs: 1,
+	}
+}
+
+func (targetSet) Run(ctx *cmd.Context, _ *cmd.Client) error {
+	err := downloadConfiguration(ctx.Args[0])
+	if err != nil {
+		return err
+	}
+	err = downloadTsuruTarget(ctx.Args[0])
+	if err != nil {
+		return err
+	}
+	fmt.Fprintln(ctx.Stdout, "Target successfully defined!")
+	return nil
+}
+
+func downloadConfiguration(server string) error {
+	url := strings.TrimRight(server, "/") + "/config.json"
+	resp, err := http.Get(url)
+	if err != nil {
+		return err
+	}
+	defer resp.Body.Close()
+	if resp.StatusCode != http.StatusOK {
+		data, _ := ioutil.ReadAll(resp.Body)
+		return fmt.Errorf("failed to download config file: %d - %s", resp.StatusCode, data)
+	}
+	return writeConfigFile(resp.Body)
+}
+
+func downloadTsuruTarget(server string) error {
```

Comment:
The piece of code used to manage targets on tsuru wasn't exported, I changed it there and updated our code. Thanks!